src/content/developers/tutorials/yellow-paper-evm/index.md

```diff
@@ -0,0 +1,396 @@
+| *I<sub>o</sub>* |  [`ORIGIN`](https://www.evm.codes/#32) | `tx.origin`
+| *I<sub>p</sub>* |  [`GASPRICE`](https://www.evm.codes/#3a)  | `tx.gasprice`
+| *I<sub>d</sub>* |  [`CALLDATALOAD`](https://www.evm.codes/#35), etc.  | `msg.data`
+| *I<sub>s</sub>* |  [`CALLER`](https://www.evm.codes/#33)  | `msg.sender`
+| *I<sub>v</sub>* |  [`CALLVALUE`](https://www.evm.codes/#34)  | `msg.value`
+| *I<sub>b</sub>* |  [`CODECOPY`](https://www.evm.codes/#39)  | `address(this).code`
+| *I<sub>H</sub>* |  Block header fields, such as [`NUMBER`](https://www.evm.codes/#43) and [`DIFFICULTY`](https://www.evm.codes/#44)  | `block.number`, `block.difficulty`, etc.
+| *I<sub>e</sub>* |  Depth of the call stack for calls between contracts (including contract creation)
+| *I<sub>w</sub>* |  Is the EVM allowed to change state, or is it running statically
+
+A few other parameters are necessary to understand the rest of section 9:
+
+| Parameter | Defined in section | Meaning |
+| --- | - | - |
+| *σ* |  2 (p. 2, equation 1) | The state of the blockchain 
+| *g* | 9.3 (p. 13)           | Remaining gas
+| *A* | 6.1 (p. 8)            | Accrued substate (changes scheduled for when the transaction ends)
+| *o* | 9.3 (p. 13)           | Output - the returned result in the case of internal transaction (when one contract calls another) and calls to view functions (when you are just asking for information, so there is no need to wait for a transaction)
+
+
+
+
+## 9.4 (Execution Overview) 
+
+Now that have all the preliminaries, we can finally start working on how the EVM works.
+
+Equations 137-142 give us the initial conditions for running the EVM:
+
+| Symbol | Initial value | Meaning |
+| - | - | - |
+| *μ<sub>g</sub>*  | *g* | Gas remaining
+| *μ<sub>pc</sub>* | *0* | Program counter, the address of the next instruction to execute
+| *μ<sub>m</sub>*  | *(0, 0, ...)* | Memory, initialized to all zeros 
+| *μ<sub>i</sub>*  | *0* | Highest memory location used 
+| *μ<sub>s</sub>*  | *()* | The stack, initially empty
+| *μ<sub>o</sub>*  | *∅*  | The output, empty set until and unless we stop either with return data ([`RETURN`](https://www.evm.codes/#f3) or [`REVERT`](https://www.evm.codes/#fd)) or without it ([`STOP`](https://www.evm.codes/#00) or [`SELFDESTRUCT`](https://www.evm.codes/#ff)). 
+ 
+ 
+Equation 143 tells us there are four possible conditions at each point in time during execution, and what to do with them:
+
+1. If *Z(σ,μ,A,I)*, it means that we have encountered an abnormal condition.
+   In that case the new state is identical to the old one (except gas gets burned)
+1. If the opcode is [`REVERT`](https://www.evm.codes/#fd), the new state is the same as the old state, some gas is lost, and we have output to return.
+1. If there is any output, (meaning we are at a [`RETURN`](https://www.evm.codes/#f3)), the state is the new state and return the output.
+1. If we aren't at one of the end conditions, continue running.
```

Comment:
Reminder to please use explicit numbering for lists in markdown on ethereum.org :-) the items are pulled out of context during translation and everything getting numbered '1' can be quite confusing for translators

```suggestion
1. If *Z(σ,μ,A,I)*, it means that we have encountered an abnormal condition.
   In that case, the new state is identical to the old one (except gas gets burned)
2. If the opcode is [`REVERT`](https://www.evm.codes/#fd), the new state is the same as the old state, some gas is lost, and we have output to return.
3. If there is any output, (meaning we are at a [`RETURN`](https://www.evm.codes/#f3)), the state is the new state and returns the output.
4. If we aren't at one of the end conditions, continue running.
```